src/vstest.console/Internal/ConsoleLogger.cs

```diff
@@ -219,10 +221,19 @@ public void Initialize(TestLoggerEvents events, Dictionary<string, string> param
                 bool.TryParse(prefix, out AppendPrefix);
             }
 
+            var ciEnvironment = Environment.GetEnvironmentVariable(ConsoleLogger.ContinuousIntegrationIndicator);
+            if (bool.TryParse(ciEnvironment, out var ci) && ci)
```

Comment:
currently only checking if true or false case insensitive.  should additional values be accepted? 1/0?